epsagon/trace.py

```diff
@@ -139,6 +142,9 @@ def initialize(
             Add an epsagon log id to all loggings and prints
         :param step_dict_output_path:
             Path in the result dict to append the Epsagon steps data
+        :param sample_rate:
+            When enabled, sampling will performed performed according to the
```

Comment:
```suggestion
            number between 0 and 1, representing the probability of each trace to be sent. When enabled (value < 1), sampling will be performed according to the
```
also, "performed performed" :)